control_panel_api/tests/test_services.py

```diff
@@ -66,21 +81,45 @@ def test_bucket_delete(self, mock_delete_bucket_policies):
 
         mock_delete_bucket_policies.assert_called()
 
+    @patch('control_panel_api.aws.create_role')
+    def test_app_create(self, mock_create_role):
+        app_slug = 'appname'
+        services.app_create(app_slug)
+
+        expected_role_name = "test_app_{}".format(app_slug)
```

Comment:
Is there a conflict here between naming conventions?  We hyphenate s3 buckets `dev-test-foo` so would it be a better idea to maintain that convention here?

Reply:
As discussed this morning via Slack, I can't remember why I didn't use hyphens (`-`) for the IAM role names. I looked again this morning and it seems like [in theory they should be legal](http://docs.aws.amazon.com/IAM/latest/UserGuide/reference_iam-limits.html) (alongside "weirder" symbols like `=` or `@` also appear to be legal...)

So...even if breaking with past (lambda functions) convention I think you're right in saying it would be nice to be consistent here...I'll change (we'll find out if it will break and there was actually a reason for not using hyphens 😱 )